### PR TITLE
appender: prepare to release 0.2.2

### PR DESCRIPTION
**Pull request by davidbarsky**
# 0.2.2 (March 17, 2022)

This release fixes an incorrect `compare_exchange` in RollingFileAppender 
when rolling files over, causing a panic.

### Fixed
- **rolling**: Fixed a panic that prohibited rolling files over. ([#1989])

[#1989]: https://github.com/tokio-rs/tracing/pull/1989

### CODE REVIEW

**Review by hawkw — CHANGES_REQUESTED** [review]
Let's make sure to also update the version number here
https://github.com/tokio-rs/tracing/blob/a5ff1afe8c8ab3fb6075b72cb4087be1ecbd5363/tracing-appender/README.md?plain=1#L19-L21
and here
https://github.com/tokio-rs/tracing/blob/a5ff1afe8c8ab3fb6075b72cb4087be1ecbd5363/tracing-appender/src/lib.rs#L125
please!

**Comment by davidbarsky** [comment]
whoops, sorry about missing some of those! your phrasing suggestions on the changelog are much better—I pulled details from the PR fixing it, but didn't fully understand all the details.

File: tracing-appender/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 0.2.2 (March 17, 2022)
+
+This release fixes an incorrect `compare_exchange` in RollingFileAppender 
+when rolling files over, causing a panic.
+
+### Fixed
+- **rolling**: Fixed a panic that prohibited rolling files over. ([#1989])
```
[the remark this answers is not in the document]
Comment (davidbarsky) [reply tracing-appender/CHANGELOG.md]: added! your phrasing is better.

**Review by hawkw — APPROVED** [review]
okay, this looks good to me --- we can ignore the CI failure on nightly, that's still because nightly ICEs.

**Pull request merged**